espnet2/tasks/enh.py

```diff
@@ -130,7 +142,13 @@
 
 decoder_choices = ClassChoices(
     name="decoder",
-    classes=dict(stft=STFTDecoder, conv=ConvDecoder, same=NullDecoder),
+    classes=dict(
+        stft=STFTDecoder,
+        conv=ConvDecoder,
+        same=NullDecoder,
+        codec=CodecDecoder,
+        seanet=SEANetEnhDecoder,
```

Comment:
Can you add an additional yaml file under `egs2/librimix/enh1/conf/tuning` for the `seanet` separator? This helps clarify how we can use the added model.